csharp/src/SammanCoaching.ParameterizedTests/Status.cs

```diff
@@ -0,0 +1,29 @@
+namespace SammanCoaching.ParameterizedTests;
+
+public enum Status
+{
+    /**
+     * Placed, but not payed yet. Still changeable.
+     */
+    PaymentExpected = 0,
```

Comment:
Why is 0 assigned here? Necessary?

Reply:
it's not necessary, i thought i saw it somewhere in java, but might be wrong.